wetterdienst/cli.py

```diff
@@ -263,32 +264,36 @@ def get_nearby(options: Munch) -> Tuple[List, List]:
     and "maximum distance in kilometers" query flavors.
 
     :param options: Normalized docopt command line options.
-    :return: nearby_stations, distances
+    :return: nearby_stations
     """
 
     nearby_baseline_args = dict(
-        latitudes=[float(options.latitude)],
-        longitudes=[float(options.longitude)],
+        latitude=float(options.latitude),
+        longitude=float(options.longitude),
+        # minimal_available_date="2020-01-01T00:00:00",
+        # maximal_available_date="2099-12-31T23:59:59",
+        minimal_available_date=datetime(2020, 1, 1),
+        maximal_available_date=datetime(2020, 1, 20),
```

Comment:
I see this might not be okay.

What do you believe we can do for a) having a deterministic outcome for the software tests but b) - more important to me - to have reasonable defaults for the average user, strongly supporting DWIM here?

Reply:
For the user, we would like to see it like that, right?
```suggestion
        minimal_available_date=datetime(2020, 1, 1),
        maximal_available_date=datetime.now(),
```

Reply:
Actually why can't we give the choice to the user? There are pretty different scenarios. I used a similar function to get stations with data from 1971 to 2000, which is a climate reference period. Ofcourse this is only one possible case which wouldn't apply for recent data.

Reply:
We could basically "mimic" the DWD classification for recent which is the last 500 days and for historical we would just take any station that has data.

Reply:
I've now used:
```
minimal_date = days500 = datetime.now() + timedelta(days=-500)
maximal_date = days30 = datetime.now() + timedelta(days=-30)
```

Just using "now" for `maximal_date` did not work. When giving dates here which does not seem to be appropriate in some kind, Pandas will croak with
```
KeyError: "None of [Int64Index([0, 0, 0, 0, 0], dtype='int64')] are in the [index]"
```

You might want to play with this in order to get an idea what I am referring to.

Now, if the current DWIM settings are working fine re. test stability, I will also be satisfied with these default settings.

Reply:
> Actually why can't we give the choice to the user?

I will be happy to consider that within a future iteration.

Reply:
Absolutely! Go ahead and merge! 

I would guess that the error with max date is related to the release cycle of the data. I think its release somewhere around 6am or so, so if you do a request between that time (midnigh to 6am) it's obviously the next day but still the file is not yet updated.

Reply:
I've now used
```
minimal_date = days500 = datetime.now() + timedelta(days=-500)
maximal_date = now = datetime.now() + timedelta(days=-2)
```
Thanks!